src/Channels/SlackWebhookChannel.php

```diff
@@ -33,15 +33,15 @@ public function __construct(HttpClient $http)
      *
      * @param  mixed  $notifiable
      * @param  \Illuminate\Notifications\Notification  $notification
-     * @return void
+     * @return \Psr\Http\Message\ResponseInterface|null
```

Comment:
@fideloper Buddy are you sure we can a `null` on this? I am not sure.

Reply:
Yes. It either returns a `ResponseInterface` or `null`.

Reply:
In practice, Guzzle will never return null (to my knowledge), but this method returned null before so it's safe if it were to return null in terms of the code being brittle.

(It was suggested by Dries in his comment above, I'll leave it up up to y'all if it matters!)

Reply:
(I commented at the same time as Dries, sorry :P )

Reply:
This is just PHP being PHP. Imo whenever a method returns something (in this case `ResponseInterface` we can consider that the other empty `return` returns `null` (which it does in reality). Otherwise if nothing explicitly is returned and only empty `return` statements are present the method can be considered `void`. If you added a return type `:void` before these changes were made it would be valid.